### PR TITLE
{vis}[GCCcore/12.3.0] Xvfb v21.1.8

### PR DESCRIPTION
**Pull request by stefan-wolfsheimer**
(created using `eb --new-pr`)
depends on #18544 #18449 #18526

### CODE REVIEW

File: easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    }),
+    ('mkfontdir', '1.0.7', {
+        'source_urls': ['https://www.x.org/archive/individual/app/'],
+        'checksums': ['bccc5fb7af1b614eabe4a22766758c87bfc36d66191d08c19d2fa97674b7b5b7'],
+    }),
+    ('bdftopcf', '1.1', {
+        'source_urls': ['https://www.x.org/archive/individual/app/'],
+        'checksums': ['699d1a62012035b1461c7f8e3f05a51c8bd6f28f348983249fb89bbff7309b47'],
+    }),
+    ('font-util', '1.3.3', {
```
Comment (branfosj) [inline easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb:68]: There is a 1.4.0 available (https://www.x.org/archive/individual/font/).

File: easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        'checksums': ['2094dd4a1ca63a61deb101d2dc618682d6e287cdbe09679502223ac445d277dc'],
+    }),
+    ('font-misc-misc', '1.1.2', {
```
Comment (branfosj) [inline easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb:71]: There is a 1.1.3 available (https://www.x.org/archive/individual/font/).

File: easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        'checksums': ['46142c876e176036c61c0c24c0a689079704d5ca5b510d48c025861ee2dbf829'],
+        'preconfigopts': local_font_misc_preconfigopts,
+    }),
+    ('xkbcomp', '1.4.6', {
+        'source_urls': ['https://www.x.org/archive/individual/app/'],
+        'checksums': ['b216a2c8c0eab83f3dc4a3d5ee2bdf7827b30e49c8907035d0f222138eca0987'],
+    }),
+    ('xkeyboard-config', '2.37', {
```
Comment (branfosj) [inline easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb:79]: There is a 2.39 available (https://www.x.org/archive/individual/data/xkeyboard-config/),

File: easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,123 @@
+easyblock = 'Bundle'
+
+name = 'Xvfb'
+version = '21.1.6'
```
Comment (branfosj) [inline easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb:4]: There is a 21.1.8 available (https://www.x.org/releases/individual/xserver/).

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb (hunk 4 above)
Comment (stefan-wolfsheimer) [reply easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb:4]: Thanks for the review. I will add more recent versions.

**Comment by stefan-wolfsheimer** [comment]
I added the new file (v21.1.8) with `--update-pr` and deleted the old version (v21.1.6) with `git rm`
This generated some clutter in the logs. Should I create a fresh pr with the correct version?

**Comment by casparvl** [comment]
> I added the new file (v21.1.8) with `--update-pr` and deleted the old version (v21.1.6) with `git rm` This generated some clutter in the logs. Should I create a fresh pr with the correct version?

To me, the current one is fine. I've seen way worse clutter than this :)

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/ff822b3cff8ca949a5aa443d802067cd for a full test report.

**Review by casparvl — APPROVED** [review]

**Comment by casparvl** [comment]
Going in, thanks @stefan-wolfsheimer!

**Pull request merged**